crud/select/compat/select.lua

```diff
@@ -75,11 +75,16 @@ local function build_select_iterator(space_name, user_conditions, opts)
         end
     end
 
+    local first = opts.first
+    if first ~= nil then
+        first = math.abs(opts.first)
+    end
```

Comment:
Probably we could declare `local first` a bit above - https://github.com/tarantool/crud/pull/155/files#diff-b78f92602289b081e8f7213e3f7f546dd3148362696b13715667d1e795262a6fR56

Also `first = math.abs(first)` seems to be better than `first = math.abs(opts.first)`.